src/test/java/org/folio/search/cql/ItemSearchFieldModifierTest.java

```diff
@@ -0,0 +1,26 @@
+package org.folio.search.cql;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.folio.search.utils.types.UnitTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+@UnitTest
+class ItemSearchFieldModifierTest {
+
+  private ItemSearchFieldModifier itemSearchFieldModifier;
+
+  @BeforeEach
+  void setUp() {
+    itemSearchFieldModifier = new ItemSearchFieldModifier();
+  }
+
+  @Test
+  void getSearchTerm_positive() {
+    var field = "items.field";
```

Comment:
```suggestion
    var field = "item.field";
```
Maybe this is a case?